draft-ietf-cdni-uri-signing.xml

```diff
@@ -1305,59 +1319,59 @@ http://*/folder/content-83112371/quality_*/segment????.mp4
         </t>
 
         <figure><artwork><![CDATA[
-  {
-    "kty": "oct",
-    "kid": "f-WbjxBC3dPuI3d24kP2hfvos7Qz688UTi6aB0hN998",
-    "use": "enc",
-    "alg": "A128GCM",
-    "k": "4uFxxV7fhNmrtiah2d1fFg"
-  }
+{
+  "kty": "oct",
+  "kid": "f-WbjxBC3dPuI3d24kP2hfvos7Qz688UTi6aB0hN998",
+  "use": "enc",
+  "alg": "A128GCM",
+  "k": "4uFxxV7fhNmrtiah2d1fFg"
+}
 ]]></artwork></figure>
 
         <t>
           JWE for client IP (aud) of [2001:db8::1/32]:
         </t>
 
         <figure><artwork><![CDATA[
-  eyJhbGciOiJkaXIiLCJraWQiOiJmLVdianhCQzNkUHVJM2QyNGtQMmhmdm9zN1F6
-  Njg4VVRpNmFCMGhOOTk4IiwiZW5jIjoiQTEyOEdDTSJ9..GfSKEqKgKROiZLWE.a
-  pcxQc3bX0Ewp9JO3VchuA.To6hoQ6rvmBvBFv85xW_5Q
+eyJhbGciOiJkaXIiLCJraWQiOiJmLVdianhCQzNkUHVJM2QyNGtQMmhmdm9zN1F6N
+jg4VVRpNmFCMGhOOTk4IiwiZW5jIjoiQTEyOEdDTSJ9..MlXN_ddGDqeZv1Om.siO
+N0_WjzXkzTT-eGLCiQg.f5g8YPO4hsQAxB7SvCc0uw
 ]]></artwork></figure>
 
         <t>
-          JWT Claim Set:
+          The JWT Claim Set before signing:
         </t>
 
         <figure><artwork><![CDATA[
-  {
-    "aud": "eyJhbGciOiJkaXIiLCJraWQiOiJmLVdianhCQzNkUHVJM2QyNGtQMm
-  hmdm9zN1F6Njg4VVRpNmFCMGhOOTk4IiwiZW5jIjoiQTEyOEdDTSJ9..GfSKEqKg
-  KROiZLWE.apcxQc3bX0Ewp9JO3VchuA.To6hoQ6rvmBvBFv85xW_5Q",
-    "exp": 1474243500,
-    "iat": 1474243200,
-    "iss": "Upstream CDN Inc",
-    "jti": "5DAafLhZAfhsbe",
-    "nbf": 1474243200,
-    "sub": "uri-regex:http://cdni.example/foo/bar/baz/[0-9]{3}.png"
-  }
+{
+  "aud": "eyJhbGciOiJkaXIiLCJraWQiOiJmLVdianhCQzNkUHVJM2QyNGtQMmh
+  mdm9zN1F6Njg4VVRpNmFCMGhOOTk4IiwiZW5jIjoiQTEyOEdDTSJ9..MlXN_ddG
+  DqeZv1Om.siON0_WjzXkzTT-eGLCiQg.f5g8YPO4hsQAxB7SvCc0uw",
+  "exp": 1474243500,
+  "iat": 1474243200,
+  "iss": "Upstream CDN Inc",
+  "jti": "5DAafLhZAfhsbe",
+  "nbf": 1474243200,
+  "sub": "uri-regex:http://cdni\\.example/foo/bar/baz/[0-9]{3}.png"
```

Comment:
Also, I just want to confirm that we need both the `\`'s? I see when I render it that they are both there, but assuming this is a JSON requirement, right?